framework/db/src/main/java/com/cloud/utils/db/GenericDaoBase.java

```diff
@@ -437,9 +437,11 @@ public List<T> searchIncludingRemoved(SearchCriteria<T> sc, final Filter filter,
             }
             return result;
         } catch (final SQLException e) {
-            throw new CloudRuntimeException("DB Exception on: " + pstmt, e);
+            logger.error("DB Exception on: " + pstmt, e);
```

Comment:
```suggestion
            logger.error("DB Exception on: {}", pstmt, e);
```
The parameterized log message can also be used in the other logs.

Reply:
can take this up later if required.